### DISCUSSION TITLE
Feature Request: Add encoding option while reading csv

### DISCUSSION THREAD

**Discussion by fuubarbaz** [duckdb/duckdb#9783] [Ideas]
Why do you want this feature?
To enable reading of UTF-16 encoded CSVs.
Something similar to what pandas read_csv function offers
`pd.read_csv("foo_data.csv"), encoding="UTF-16")`

**Comment by minikinl** [comment]
The specified encoding format also needs to be supported when exporting

🙏🙏🙏🙏

```sql
COPY (...) TO 'export.csv' (ENCODING 'utf8')
```

**Comment by nshiab** [comment]
That would be great. Is there any plan to add it to DuckDB?

**Comment by lkuich** [comment]
We're also running into this issue 🙏

**Comment by galloloris** [comment]
Running into this issue as well.

**Comment by fredguth** [comment]
Reading and writing.
